templates/partners/ihv-and-oem.html

```diff
@@ -6,149 +6,218 @@
-        linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
-        linear-gradient(216deg, #E95420 0%, #5E2750 33%, #2C001E 71%);
-      background-repeat: no-repeat;
-      background-size: 105% 5%, 70% 100%, 100% 40%, 100% 100%;
-      background-position: bottom -1px left, top right, top left;
-    }
-  }
-</style>
 
-<section class="p-strip is-deep">
-  <div class="u-fixed-width u-align--center">
+<section class="p-section">
+  <div class="row p-block">
+    <hr class="p-rule">
+    <h2>A selection of our IHV and OEM partners</h2>
+  </div>
+  <div class="row">
     <div class="p-logo-section">
-      <p class="p-muted-heading u-no-max-width">A selection of our IHV and OEM partners</p>
-      <div class="p-logo-section__items">
-        {% for partner in partners %}
-          <div class="p-logo-section__item">
-            <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
-          </div>
-        {% endfor %}
+
+      <div class="p-logo-section__items u-no-margin--bottom">
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/9c7c5cbe-lenovo-logo.png" alt="Lenovo logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/f546f1d4-dell-logo.png" alt="Dell logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/ba9ca259-qct-logo.png" alt="QCT logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/c9b393e3-supermicro-logo.png" alt="Supermicro logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/460095c1-nec-logo.png" alt="Nec logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/d8e44af4-huawei-logo.png" alt="Huawei logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/d8dc6545-hp-enterprise-logo.png" alt="HPE logo">
+        </div>
+        <div class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/6522f0f5-ericsson-logo.png" alt="Ericsson logo">
         </div>
       </div>
-      <p class="u-align-text--center"><a href="/partners/find-a-partner?filters=ihv-oem">See all IHV/OEM partners</a></p>
     </div>
+  </div>
+  <div class="row">
+    <div class="col-6 col-start-large-7 col-medium-3 col-start-medium-4">
+      <p><a href="/partners/find-a-partner?filters=ihv-oem">See all IHV/OEM partners</a></p>
```

Comment:
This is just my opinion, but I think this looks better on the original design. This positioning looks a bit strange

Reply:
@ClementChaumel rightly pointed out that this page had way too many `<hr>`s, so I decided to omit a few - without an hr above this bit, it needs to sit closer to the central keyline - otherwise there's nothing to guide the eye to it.